src/Actions/Commands/Quit.ts

```diff
@@ -2,9 +2,9 @@ import * as vscode from 'vscode';
 import { Command } from './Base';
 
 export class QuitCommand extends Command {
-
     execute(): Thenable<undefined> {
-        return vscode.commands.executeCommand('workbench.action.closeActiveEditor');
+        return vscode.commands.executeCommand(
+            'workbench.action.closeActiveEditor',
+        );
```

Comment:
It seems there are quite a few changes like this, which might be due to the default print width being rather small. Did you consider using something wider than the default, e.g. 100?

Reply:
Ah no, I just stuck with the default width to be honest. Happy to do 100 - do you have a strong preference? :)

Reply:
Personally i don't care too much - i think consistency is the main thing.

However, because this is TypeScript and a bit more verbose than JavaScript, there are a few indentations that look a bit awkward to me in this PR. The one commented above is okay, but this one might be easier to read with a slightly longer line wrap:
```typescript
            const thenable: Thenable<
                boolean | undefined
            > = commands.executeCommand('cursorPageDownSelect');
```

Reply:
I'm not very familiar with Prettier. Is there an option to make it look like this?
```
const thenable: Thenable<boolean | undefined> =
    commands.executeCommand('cursorPageDownSelect');
```

Reply:
I think a higher print width will help here - I've gone for 100. It's easily changed later.